spec/dry/types/definition_spec.rb

```diff
@@ -16,33 +16,37 @@
   end
 
   describe '#try' do
-    subject(:try) { type.try(value) }
-
-    let(:success) { try.success?    }
-    let(:failure) { try.failure?    }
-    let(:input)   { try.input       }
+    let(:result) { type.try(value) }
 
     context 'when given valid input' do
       let(:value) { 'foo' }
 
-      specify { expect(success).to be(true)  }
-      specify { expect(failure).to be(false) }
-      specify { expect(input).to be(value)   }
+      it 'returns a success' do
+        expect(result).to be_success
```

Comment:
I opted to not use `be_success` before since it asserts truthyness and not that `success?` returns `true`